src/handlers/index.ts

```diff
@@ -269,6 +270,44 @@ export async function postBlockPacketHook(ctx: Context, entities: Entities) {
   }
 }
 
+async function postBlockFeeHook(ctx: Context, entities: Entities) {
+  let updatedSendPackets: SendPacket[] = [];
+  let updatedSendPacketFees: SendPacketFeeDeposited[] = [];
+
+  for (let sendPacketFee of entities.sendPacketFees) {
+    let sendPacket = await ctx.store.findOne(SendPacket,
+      {
+        where: {
+          chainId: sendPacketFee.chainId,
+          srcChannelId: sendPacketFee.channelId,
+          sequence: sendPacketFee.sequence
+        }
+      });
+    if (sendPacket) {
+      sendPacket.totalRecvFeesDeposited = sendPacket.totalAckFeesDeposited + BigInt(sendPacketFee.recvGasLimit * sendPacketFee.recvGasPrice);
+      sendPacket.totalAckFeesDeposited = sendPacket.totalAckFeesDeposited + BigInt(sendPacketFee.ackGasLimit * sendPacketFee.ackGasPrice);
```

Comment:
might be nice to have a `sendPacket.mostRecentGasLimit` property as well that the packet sentry can use, so it doesn't have to query for all feesDeposited events and always take the last one